Mars_Onboarding_Specflow/StepDefinitions/SkillStepDefinition.cs

```diff
@@ -39,6 +39,7 @@ public void GivenNavigatesToSkillsTabInProfilePage()
         public void WhenUserEntersSkillAndSkillLevel(string skill, string skilllevel)
         {
             SkillTab skillTabObj = new SkillTab();
```

Comment:
you should make use of constructor and create a skillTab once no point creating it again 

inside constructor 
private SkillTab skillTabObj;

public ClassName(){

 skillTabObj = new SkillTab();

}

Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors